README.md

```diff
@@ -194,6 +194,10 @@ Here is a list of plugin we have successfully tested to support configuration-as
  - [x] warnings-plugin (>= 4.66) ([details](demos/warnings/README.md))
  - [x] kubernetes plugin ([details](demos/kubernetes/README.md))
  - [ ] more to come soon...
+ 
+### Known compatibility issues
+
+[Plugins compatibility with Jenkins Configuration-as-Code plugin](https://wiki.jenkins.io/display/JENKINS/Plugins+compatibility+with+Jenkins+Configuration-as-Code+plugin) is being used to track known compatibility issues.
```

Comment:
Maybe I will have to switch to a dashboard, TBD

Reply:
that's ok, I don't expect to be done with docs before next week and let you know before I want to merge so we can see what's the status of the dashboard